R/vector_parameters.R

```diff
@@ -53,6 +53,12 @@ set_species <- function(parameters, species, proportions) {
   if (length(species) != length(proportions)) {
     stop('You must give proportions for each species')
   }
+  if (any(proportions <= 0)) {
```

Comment:
Shouldn't this be if `(any(proportions < 0))`
We can specify 0 proportions right?